core/templates/dev/head/components/forms/schema_editors/schema_based_bool_editor_directive.html

```diff
@@ -1,4 +1,4 @@
-<div class="input-group">
+<div class="input-group" style="float:left;margin-right:4px">
```

Comment:
This is a generic component that is used everywhere. You want to float-left based on a specific context. So the float-left styling should be used only in that specific context (perhaps in the wrapper div).